src/i18n/strings/en_EN.json

```diff
@@ -3170,6 +3175,34 @@
     "Submit": "Submit",
     "Something went wrong in confirming your identity. Cancel and try again.": "Something went wrong in confirming your identity. Cancel and try again.",
     "Start authentication": "Start authentication",
+    "The linking wasn't completed in the required time.": "The linking wasn't completed in the required time.",
+    "The scanned code is invalid.": "The scanned code is invalid.",
+    "Linking with this device is not supported.": "Linking with this device is not supported.",
+    "The request was declined on the other device.": "The request was declined on the other device.",
+    "The other device is already signed in.": "The other device is already signed in.",
+    "The other device isn't signed in.": "The other device isn't signed in.",
+    "The request was cancelled.": "The request was cancelled.",
+    "An unexpected error occurred.": "An unexpected error occurred.",
+    "The homeserver doesn't support signing in another device.": "The homeserver doesn't support signing in another device.",
+    "Devices connected": "Devices connected",
+    "Check your mobile device, the code below should be displayed. Confirm that the code below matches with that device:": "Check your mobile device, the code below should be displayed. Confirm that the code below matches with that device:",
+    "No match?": "No match?",
+    "Confirm that the code below matches with your mobile device:": "Confirm that the code below matches with your mobile device:",
+    "Please ensure that you know the origin of this code. By linking devices, you will provide someone with full access to your account.": "Please ensure that you know the origin of this code. By linking devices, you will provide someone with full access to your account.",
+    "Scan the QR code below with your device that’s signed out.": "Scan the QR code below with your device that’s signed out.",
+    "Start at the sign in screen": "Start at the sign in screen",
+    "Select ‘Scan QR code’": "Select ‘Scan QR code’",
```

Comment:
I can't remember, but I don't think we're supposed to use magic quotes in strings - check with design to see if they care.

Reply:
Ironically the magic quotes came from them...

Reply:
Yup, it's on all of the designs but it's been declared a non-feature in the past (just that their operating system isn't worth fighting to get rid of them at design time)